typescript/introduction/tsc-variable-declaration/tsc-let.md

```diff
@@ -0,0 +1,91 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Introducing `let`
+
+---
+## Content
+
+Using `var` can sometimes lead to unintuitive code or unexpected bugs. With `let` we can fix those problems.
```

Comment:
we should emphasise that this is per the new ES spec which typescript adheres to